xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/CommentsTab.java

```diff
@@ -173,4 +173,15 @@ public boolean hasEditButtonForCommentByID(int commentId)
         return getDriver().findElementsWithoutWaiting(By.xpath("//div[@id='xwikicomment_"
             + commentId + "']//a[contains(@class, 'edit')]")).size() > 0;
     }
+
+    /**
+     * @param commentId the comment id
+     * @return true if the comment has the delete button
+     * @since 10.6RC1
+     */
+    public boolean hasDeleteButtonForCommentByID(int commentId)
+    {
+        return getDriver().findElementsWithoutWaiting(By.xpath("//div[@id='xwikicomment_"
+            + commentId + "']//a[contains(@class, 'delete')]")).size() > 0;
```

Comment:
I'm in favor of using the CSS selector when it's simpler

    "#xwikicomment_" + commentId + " a.delete"